### ISSUE TITLE
multiple methods defined twice in Mixture

### ISSUE THREAD

**Issue by zoltuz** [BuildACell/BioCRNPyler#151]
![Screenshot 2020-08-01 at 20 46 51](https://user-images.githubusercontent.com/17138838/89109342-3d4dfe00-d438-11ea-9c0d-aadfb91ca361.png)

**Comment by WilliamIX** [comment]
There is a difference. mixture.mechanisms (a dictionary) versus getting and adding a single mechanism inside mixture.mechanisms

**Comment by WilliamIX** [comment]
As noted above, this is not a bug, but is by design. By wrapping a dictionary (global_mechanisms) as a property, I had to make a helper function to add items to that dictionary (add_global_mechanism). The setter resets the entire dictionary. Note that the same thing is done with mechanisms and Components and something similar will be done with Species and Reactions in ChemicalReactionNetwork later. If there are problems with this solution, lets discuss. @zoltuz

**Comment by zoltuz** [comment]
![Screenshot 2020-09-04 at 19 27 24](https://user-images.githubusercontent.com/17138838/92274374-02cafb80-eee5-11ea-8f75-2ea72146f073.png)
These functions are defined twice, it looks like there are duplicates
```python
get_mechanism() # line 135 vs line 295
global_mechanisms()# line 149 vs 309 
add_global_mechanism() # line 165 vs 323
```

**Comment by WilliamIX** [comment]
Ahh I see - that is so strange! must have happened via  merge somewhere by accident. Fixed in my newest PR.

**Comment by zoltuz** [comment]
I guess so. I'll double-check it in the PR

**Comment by zoltuz** [comment]
This issue has been solved in PR #180
